api/types/descriptor/descriptor.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package containerd.v1.types;
+
+import "gogoproto/gogo.proto";
+
+message Descriptor {
```

Comment:
I saw that this is matching the oci image spec. But could we get a better name? Or at least a description?

Maybe `LayerDescriptor`?